Source/dolittle-create-readmodel.js

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env node
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Dolittle. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import args from 'args';
+import global from './global';
+
+args
+    .example("dolittle create readmodel [name] [namespace]", "Creates a read model with a given name and namespace in the current folder");
```

Comment:
Namespace should be deducted. 
Remember; namespace is a C# feature - and we can figure out what it is. As mentioned in the issue: https://github.com/dolittle-tools/cli/issues/8

Lets walk through a bit on context and what the thinking has been.